src/checksync.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
This is the JSON Schema definition file against which our config files will be compared.